metrics/grafana/tikv_summary.json

```diff
@@ -500,6 +508,14 @@
               "legendFormat": "{{instance}}",
               "refId": "A",
               "step": 10
+            },
+            {
+              "expr": "tikv_server_memory_quota_bytes{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", job=~\".*tikv\"}",
```

Comment:
should it be added to `tikv_details` as well?

Reply:
> should it be added to `tikv_details` as well?

Done.